sbc_compassion/models/import_letter_line.py

```diff
@@ -40,8 +42,8 @@ class ImportLetterLine(models.Model):
 
     status = fields.Selection([
         ("lang", _("Error in Language")),
-        ("sponsor", _("Error in Sponsorship")),
-        ("encourager", _("Sponsorship not found")),
+        ("sponsorship", _("Sponsorship not Found")),
+        ("person", _("Error in Partner or Child")),
```

Comment:
I think I should rename it to "Partner or child not found". It is in order to have different errors depending on where the error is for the sponsorship. The two mains problems can be that the child or the partner is not found and the other one is that there is no sponsorship between them